console/cdvdcontrol/cdvdcontrol.cpp

```diff
@@ -104,148 +102,132 @@
-    if (isPlextorLockPresent(dev) != 0) {
+    if (isPlextorLockPresent(dev))
       dev->ven_features |= PX_ERASER;
-    }
-    if (yamaha_check_amqr(dev) == 0) {
+    if (!yamaha_check_amqr(dev))
       dev->ven_features |= YMH_AMQR;
-    }
-  } else if (isYamaha(dev) != 0) {
-    if (yamaha_check_amqr(dev) == 0) {
+  } else if (isYamaha(dev)) {
+    if (!yamaha_check_amqr(dev))
       dev->ven_features |= YMH_AMQR;
-    }
-    if (yamaha_check_forcespeed(dev) == 0) {
+    if (!yamaha_check_forcespeed(dev))
       dev->ven_features |= YMH_FORCESPEED;
-    }
-    if (yamaha_f1_get_tattoo(dev) == 0) {
+    if (!yamaha_f1_get_tattoo(dev))
       dev->ven_features |= YMH_TATTOO;
-    }
-  } else if (isPioneer(dev) != 0) {
-    if (pioneer_get_quiet(dev) == 0) {
+  } else if (isPioneer(dev)) {
+    if (!pioneer_get_quiet(dev))
       dev->ven_features |= PIO_QUIET;
-    }
   }
 
-  //  printf("Trying opcode E9 modes...\n");
-  //  for (int i=0; i<256; i++) {if (!plextor_get_mode(dev,i)) printf(" MODE
-  //  0x%02X\n",i);}
+  //	printf("Trying opcode E9 modes...\n");
+  //	for (int i=0; i<256; i++) {if (!plextor_get_mode(dev,i)) printf(" MODE
+  // 0x%02X\n",i);}
 
-  //  printf("Trying opcode ED modes...\n");
-  //  for (int i=0; i<256; i++) {if (!plextor_get_mode2(dev,i)) printf(" MODE
-  //  0x%02X\n",i);}
+  //	printf("Trying opcode ED modes...\n");
+  //	for (int i=0; i<256; i++) {if (!plextor_get_mode2(dev,i)) printf(" MODE
+  // 0x%02X\n",i);}
 
-  if ((flags & FL_SUPPORTED) != 0U) {
-    //      printf("____________________________\n");
-    cout << "\n** Supported features:\n";
-    cout << "AudioMaster Q.R.    : "
-         << (((dev->ven_features & YMH_AMQR) != 0U) ? "YES" : "-") << "\n";
+  if (flags & FL_SUPPORTED) {
+    //		printf("____________________________\n");
+    printf("\n** Supported features:\n");
+    printf("AudioMaster Q.R.    : %s\n",
+           dev->ven_features & YMH_AMQR ? "YES" : "-");
     printf("Yamaha ForceSpeed   : %s\n",
-           ((dev->ven_features & YMH_FORCESPEED) != 0U) ? "YES" : "-");
+           dev->ven_features & YMH_FORCESPEED ? "YES" : "-");
     printf("Yamaha DiscT@2      : %s\n",
-           (dev->ven_features & YMH_TATTOO) ? "YES" : "-");
+           dev->ven_features & YMH_TATTOO ? "YES" : "-");
     printf("Hide CD-R           : %s\n",
-           (dev->ven_features & PX_HCDRSS) ? "YES" : "-");
+           dev->ven_features & PX_HCDRSS ? "YES" : "-");
     printf("SingleSession       : %s\n",
-           (dev->ven_features & PX_HCDRSS) ? "YES" : "-");
+           dev->ven_features & PX_HCDRSS ? "YES" : "-");
     printf("SpeedRead           : %s\n",
-           (dev->ven_features & PX_SPDREAD) ? "YES" : "-");
+           dev->ven_features & PX_SPDREAD ? "YES" : "-");
     printf("PoweRec             : %s\n",
-           (dev->ven_features & PX_POWEREC) ? "YES" : "-");
+           dev->ven_features & PX_POWEREC ? "YES" : "-");
     printf("GigaRec             : %s\n",
```

Comment:
Codacy found an issue: [Clarify calculation precedence for '&' and '?'.](https://app.codacy.com/manual/jeremiahwilson777/qpxtool/pullRequest?prid=6086205)